src/helpers.js

```diff
@@ -486,3 +487,29 @@ export const isAnchoring = (
   );
   return timeDiffMinutes >= 0 && timeDiffMinutes < anchoringDuration;
 };
+/**
+ * Function to format supported domains in the format for Select/Dropdown
+ * components
+ * @param {array} supporteddomains
+ */
+export const getContractorDomains = (supporteddomains) => [
+  { label: "No domain defined", value: 0 },
+  ...supporteddomains.map((item) => ({
+    label: capitalize(item.description),
+    value: item.type
+  }))
+];
+
+/**
+ * Function to return the domain name given an array of contractor domains and
+ * a domain type
+ * @param {array} contractorDomains
+ * @param {number|string} op
+ */
+export const getDomainName = (contractorDomains, op) => {
+  let domainName;
+  contractorDomains.forEach((domain) => {
+    if (domain.value === op) domainName = domain.label;
+  });
```

Comment:
Liked this approach, but I would use:
```suggestion
  const { label: domainName } = contractorDomains.find(
    (domain) => domain.value === op
  );
```